components/serializer.rst

```diff
@@ -897,7 +897,7 @@ Option                  Description                                           De
 ``csv_escape_char``     Sets the escape character (at most one character)
 ``csv_key_separator``   Sets the separator for array's keys during its        ``.``
                         flattening
-``csv_headers``         Sets the headers for the data                         ``[]``, inferred from input data's keys
+``csv_headers``         Sets the order of the header and data columns         ``[]``, inferred from input data's keys
```

Comment:
I think I understand the problem you want to fix ... but I'm not sure the fix is correct.

If I read "sets the headers for the data", I think about this:

```php
'csv_headers' => ['Header 1 Name', 'Header 2 Name', '...']
```

But if I read "sets the order of the header and data", I understand it like this:

```php
# header name => header position
'csv_headers' => ['Header 1 Name' => 2, 'Header 2 Name' => 1, '...']
```

Maybe adding a small example could help?

Reply:
Thank you for your comments. I added an example, hopefully that will make it more clear.